README.md

```diff
@@ -54,6 +54,50 @@ pyyaml
 urllib3
 ```
 
+### Developer setup
+
+Clone the repository using `git`:
+```
+git clone https://github.com/zowe/zowe-client-python-sdk.git
+```
+
+We recommend that developers create a virtual environment to install all the required dependencies for this project.  
+**Note:** Use of `venv`, the virtual environment module, requires at least version 3.3 of Python.
```

Comment:
The Zowe Python SDK itself requires Python 3.7 or newer (because it uses features added in 3.7 like dataclasses).

Would it make sense to add a note about this higher up in the readme, and then this one about venv would no longer be needed?